### PR TITLE
overhaul website theme, add responsive styling, navbar animations

### PR DESCRIPTION
**Pull request by KendrickAng**
- switch theme colors to shades of grey (pun intended)
- split landing page into project and about me page
- add 'highlight' navbar animations
- responsive styling for screens smaller than 480px

### CODE REVIEW

File: portfolio/src/main/webapp/origami.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// number of images in the origami folder, hardcoded :(
+const N = 20;
```
Comment (WYHNUS) [inline portfolio/src/main/webapp/origami.js]: Nit: `NUMBER_OF_IMAGES` might be a better name.

File: portfolio/src/main/webapp/origami.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// number of images in the origami folder, hardcoded :(
+const N = 20;
+const PATH_PREFIX = 'images/origami/';
+const CARD_CN = 'gallery__card';
+
+function loadOrigamiToGallery() {
+    const gallery = document.getElementById('gallery');
+    // 1-indexed for easier reference
```
Comment (WYHNUS) [inline portfolio/src/main/webapp/origami.js:8]: Nit: we normally use 0 based index, but it's up to you to decide. :)

**Review by WYHNUS — APPROVED** [review]

**Review by WYHNUS — APPROVED** [review]

File: portfolio/src/main/webapp/origami.js [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
-    for (let i = 1; i <= N; i++) {
+    for (let i = 0; i < NUMBER_OF_IMAGES; i++) {
         const card = document.createElement('div');
         card.className = CARD_CN;
 
         const img = document.createElement('img');
-        const src = i < 10
-            ? `${PATH_PREFIX}00${i}-${origami[i]}.svg`
-            : `${PATH_PREFIX}0${i}-${origami[i]}.svg`;
+        const src = i < 9
```
Comment (WYHNUS) [inline portfolio/src/main/webapp/origami.js:20]: Nit: avoid magic number, and consider declare it at the top of file as a constant as well.

**Review by WYHNUS — APPROVED** [review]

File: portfolio/src/main/webapp/origami.js (hunk 3 above)
Comment (KendrickAng) [reply portfolio/src/main/webapp/origami.js:20]: good idea, will do that!

**Pull request merged**